### ISSUE TITLE
No subfolders for sidebar menu items

### ISSUE THREAD

**Issue by rfparedes** [tomjohnson1492/documentation-theme-jekyll#129]
Is there anyway to have a sidebar menu item without any subfolders? So instead of clicking a parent menu item and opening up the submenu items underneath, clicking the menu item will take you right to a page.

**Comment by tomjoht** [comment]
yes, I'll add this feature in an upcoming version, as some have asked for it. It's not a hard change to make. Thanks for the tip.
